api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -8,10 +8,20 @@
 ENVIRONMENTS = {
   "all-of-us-workbench-test" => {
     :publisher_account => "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com",
-    :source_cdr_project => "all-of-us-ehr-dev",
-    :ingest_cdr_project => "fc-aou-vpc-ingest-test",
-    :dest_cdr_project => "fc-aou-cdr-synth-test",
-    :config_json => "config_test.json"
+    # hardcoded for prototype
+    :config_json => "config_local.json",
+    :accessTiers => {
+      "registered" => {
+        :source_cdr_project => "all-of-us-ehr-dev",
+        :ingest_cdr_project => "fc-aou-vpc-ingest-test",
+        :dest_cdr_project => "fc-aou-cdr-synth-test",
+      },
+      "tier2" => {
+        :source_cdr_project => "all-of-us-ehr-dev",
+        :ingest_cdr_project => "fc-aou-cdr-ingest-test-2",
```

Comment:
nit: should this be `fc-aou-vpc-ingest-test-2` to correspond with the registered tier ingest project

Reply:
I requested a project of the name you suggested from the DRC but something got lost in translation, so what's recorded here is what was actually created.

We *could* go back and rename.  I dunno.

Reply:
Eh, probably nbd in test